apps/forum-app/src/backend/controllers/AnswerController.js

```diff
@@ -148,14 +148,11 @@ export const deleteAnswerHandler = function (schema, request) {
         }
       );
     }
-    const question = this.db.questions.findBy({ _id: questionId });
+    const question = schema.questions.findBy({ _id: questionId }).attrs;
     const answerIndex = question.answers.findIndex(
       (answer) => answer._id === answerId
     );
-    if (
-      question.answers[answerIndex].username !== user.username &&
-      question.username !== user.username
-    ) {
```

Comment:
Hi @sohamsshah 
Could you help me understand this logic?
Does this mean that a user who has created this question can delete the answer provided by some other user?

Reply:
Yes, precisely!